app/uk/gov/hmrc/agentepayeregistration/audit/AuditService.scala

```diff
@@ -46,16 +47,33 @@ class AuditService @Inject()(val auditConnector: AuditConnector) {
       Seq("payeAgentRef" -> agentReference.value,
         "agentName" -> registrationRequest.agentName,
         "contactName" -> registrationRequest.contactName,
-        "telephoneNumber" -> registrationRequest.telephoneNumber.getOrElse(""),
+        "phoneNo" -> registrationRequest.phoneNo.getOrElse(""),
```

Comment:
in the story description, there is `telephoneNumber` rather than `phoneNo`

Reply:
yes but in the API documentation it's phoneNo and email...? https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?spaceKey=AG&title=%231337+Create+Agent+Known+Facts 
I thought changing that might affect the API? Or does it not make a difference?

Reply:
it will not affect API, this is local to audit event only